src/lib/chips/chip.ts

```diff
@@ -110,14 +142,110 @@ export class MdChip extends _MdChipMixinBase implements Focusable, OnDestroy, Ca
     this.onFocus.emit({chip: this});
   }
 
+  /**
+   * Allows for programmatic removal of the chip. Called by the MdChipList when the DELETE or
+   * BACKSPACE keys are pressed.
+   *
+   * Note: This only informs any listeners of the removal request, it does **not** actually remove
```

Comment:
Would slightly rephrase this part of the comment as 
"Informs any listeners of the removal request. Does not remove the chip from the DOM."